screens/NewHookScreen/NewHookScreen.js

```diff
@@ -84,11 +127,53 @@ const NewHookScreen = (props) => {
             />
             <Icon name="angle-down" style={styles.icon} size={22} color={colors.darkBlue} />
           </View>
+
+          {(hookType === 'SlackHook' && hookOf === 'output') && (<><Text style={styles.textHeader}>Entidades</Text>
+            <View style={styles.pickerContainer}>
+              <RNPickerSelect
+                value={reference}
+                onValueChange={slackReferenceHandler}
+                placeholder={{}}
+                items={slackEntities.map((item) => {
+                  if ('purpose' in item) {
```

Comment:
ojo que `in` busca la key en todas las propiedades del objeto, incluyendo las heredadas del prototype. En este caso asumo que con "purpose" no habría problema pero pa que lo tengan presente

Reply:
se tendrá presente. gracias!